tests/multi_ptr/multi_ptr_prefetch_member_core.cpp

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests multi_ptr prefetch member for core types
+//
+*******************************************************************************/
+
+#include "../common/type_coverage.h"
+#include "multi_ptr_common.h"
+#include "multi_ptr_prefetch_member.h"
+
+namespace multi_ptr_prefetch_member_core {
+
+TEST_CASE("Prefetch member. core types", "[multi_ptr]") {
+  using namespace multi_ptr_prefetch_member;
+  auto types = multi_ptr_convert::get_types();
```

Comment:
It probably doesn't, but does this contain the `void` type? Because `multi_ptr<void>` doesn't have the `prefetch` operation, so `void` shouldn't be included here.

Reply:
Verification, that current data type isn't void has been added in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/352/commits/e2c843f9c2d8656707a1d45d57adddc3ca994b9e